src/less/components/city-chooser.less

```diff
@@ -5,12 +5,15 @@
 
 .cities {
   &:extend(.flex-row);
-  padding: 20px;
+  flex-wrap: wrap;
+  justify-content: center;
+  padding: 3vw;
 }
 
 .city {
   &:extend(.flex-column, .centered);
-  margin: 0 5vw;
+  width: 100px;
```

Comment:
https://plugins.jetbrains.com/plugin/7489-css-alphabetical-rearranger
I recommend it :D